cmd/svcat/svcat_test.go

```diff
@@ -41,17 +41,100 @@ import (
 	"github.com/kubernetes-incubator/service-catalog/internal/test"
 	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
 	"github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset/fake"
+	svcatfake "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset/fake"
 	"github.com/kubernetes-incubator/service-catalog/pkg/svcat"
 	"github.com/kubernetes-incubator/service-catalog/pkg/svcat/service-catalog"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"gopkg.in/yaml.v2"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	k8stesting "k8s.io/client-go/testing"
 )
 
 var catalogRequestRegex = regexp.MustCompile("/apis/servicecatalog.k8s.io/v1beta1/(.*)")
 var coreRequestRegex = regexp.MustCompile("/api/v1/(.*)")
 
+// Verify that svcat gracefully handles when the namespaced broker feature flag is disabled
+// TODO: Once we take Namespaced brokers out from behind the feature flag, this test won't be necessary
```

Comment:
probably once we get it out from the feature flag + some number of versions?

Reply:
Yeah I think you are right, I'm just not sure how long yet. Maybe until 1.0 honestly...